orm/src/main/java/org/hibernate/search/batchindexing/impl/IdentifierProducer.java

```diff
@@ -105,9 +105,18 @@ private void inTransactionWrapper(StatelessSessionImpl upperSession) throws Exce
 		}
 		try {
 			Transaction transaction = session.accessTransaction();
-			transaction.begin();
-			loadAllIdentifiers( session );
-			transaction.commit();
+			final boolean controlTransactions = ! transaction.isActive();
```

Comment:
We do this regularly, to make sure the `!` is visible.
We simply decided to not enforce it on checkstyle but I still prefer it.

Sometimes to highlight it I might even write `( x == false )`